runtime/doc/tabpage.txt

```diff
@@ -61,16 +61,25 @@ In the GUI tab pages line you can use the right mouse button to open menu.
 :[count]tabe[dit]				*:tabe* *:tabedit* *:tabnew*
 :[count]tabnew
 		Open a new tab page with an empty window, after the current
-		tab page.  For [count] see |:tab| below.
+		tab page.  If [count] is given the new tab page appears after 
+		the tabpage [count] otherwise the new tab page will appear
+		after the current one. >
+		    :tabnew     " opens tabpage after the current one
```

Comment:
I've never liked how in the vim docs, indented blocks such as this aren't separated between the above paragraph, like this:
```
after the current one. >

		    :tabnew     " opens tabpage after the current one
```

What do you think?